packages/ace-axe-runner-puppeteer/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@daisy/ace-axe-runner-puppeteer",
```

Comment:
ok, so this is a very minor detail, but "axe-runner" might not be the most accurate name, as we're not only running axe but also other things like outline and data extractions, etc.
I can't really suggest any better name though (`browser-checks-runner` sounds a bit mouthful), so works for me…

Reply:
This is why it's called the "ace axe" runner ;)
(another way to read the name)